doc/sphinx/package_service_mapping.json

```diff
@@ -641,6 +655,20 @@
     ],
     "service_name": "PostgreSQL"
   },
+  "azure-mgmt-rdbms": {
+    "category": "Management",
+    "namespaces": [
+      "azure.mgmt.rdbms"
+    ],
+    "service_name": "RDBMS"
```

Comment:
Nit: Is this for open source one? I think if it is, then this would be the name: `OSS RDBMS `

Reply:
I love all this feedback. Making adjustments.

Reply:
Not completely sure though - whenever I search it pops up Azure SQL Database

Reply:
I think I'll stick with OSS RDBMS. I have another one that's actually sql